vagrant/config/boxes.ci.yaml

```diff
@@ -0,0 +1,17 @@
+operating_systems:
+  - centos7
+  - debian9
+  - debian10
+  - ubuntu1604
+  - ubuntu1804
+
+foreman_versions:
```

Comment:
In CI we always know which OS + version we're testing. CI also relies on the pipelines which AFAIK always defines its own boxes. So we can actually generate the `operating_systems` on the fly in Jenkins and leave out all `foreman_versions`.